pages/userSettings/userSettings.js

```diff
@@ -37,34 +37,69 @@ router.post('/', (req, res, next) => {
             if (ERR(err, next)) return;
             res.redirect(req.originalUrl);
         });
-    } else {
+    }
+    else if(req.body.__action === 'update_settings'){
+        if('dark_mode' in req.body){
+            var params = {
+                user_id: res.locals.authn_user.user_id,
+                dark_mode: 1,
```

Comment:
Is this parameter a binary switch (`darkmode = 0/1`) or a theme number? Sometimes we seem to be doing one and sometimes the other.